.github/workflows/dapr-test-sdk.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         if: env.PR_NUMBER != ''
         uses: artursouza/sticky-pull-request-comment@v2.2.0
         with:
-          header: ${{ github.run_id }}
+          header: ${{ github.run_id }}-python
```

Comment:
This allows having a separate comment for python and java, so updates should work now (hopefully)